src/features.rs

```diff
@@ -140,8 +140,6 @@ macro_rules! rust_feature_def {
 rust_feature_def!(
     /// Untagged unions ([RFC 1444](https://github.com/rust-lang/rfcs/blob/master/text/1444-union.md))
     => untagged_union;
-    /// Constant function ([RFC 911](https://github.com/rust-lang/rfcs/blob/master/text/0911-const-fn.md))
-    => const_fn;
```

Comment:
I think `const fn` was also needed for SpiderMonkey, so not sure we can get away without it so easily...

Reply:
@emilio I'll leave it out for now because it would be easy to add back in. Certainly the bitfield allocation unit constructors have no hope of being `const fn`.